frontend/src/theme/index.tsx

```diff
@@ -4,7 +4,7 @@ import "@fontsource/dm-sans";
 
 import { extendTheme } from "@chakra-ui/react";
 
-import PasswordInputFieldStyle from "./PasswordInputFieldStyle";
+import PasswordInputFieldStyle from "./Components/PasswordInputFieldStyle";
```

Comment:
Note: We should organize all our custom component styles inside a separate folder so they don't get messy. Thoughts?